server/pulp/server/managers/status.py

```diff
@@ -51,9 +54,14 @@ def get_broker_conn_status():
     # explicit connection and then release it.
     # See https://github.com/celery/kombu/issues/432 for more detail.
     try:
-        conn = celery.connection()
         conn.connect()
+        transport = conn._transport
         conn.release()
+
+        # An explicit call to __del__ to release file descriptors
+        # https://pulp.plan.io/issues/2124#note-9
+        transport.__del__()
```

Comment:
One thought that *just* occured to me, though, is that if upstream takes `__del__` away, then we'll `AttributeError` here. It is work guarding this call against the most likely upstream change, which is hopefully to stop doing anything that depends on `__del__` being called? :)